bundler.d/fog.rb

```diff
@@ -1,3 +1,3 @@
 group :fog do
-  gem 'fog', '1.42.1', :require => false
+  gem 'fog', '2.1.0', :require => false
```

Comment:
Do we actually still need fog itself if we have fog-core and all the various fog-* providers?